src/types.rs

```diff
@@ -1225,7 +1225,7 @@ pub enum NodePreference {
 
 impl Default for NodePreference {
     fn default() -> Self {
-        NodePreference::Random
+        NodePreference::Leader
```

Comment:
**FIX** Could you include this change in the PR description so we have this in the changelog? This is important, potentially breaking change.